Source/DataSources/BillboardVisualizer.js

```diff
@@ -50,6 +50,7 @@ define([
     var EntityData = function(entity) {
         this.entity = entity;
         this.billboard = undefined;
+        this.textureValue = undefined;
```

Comment:
It's a shame we have to keep a reference to this.  This is something we avoid like the plague in the lower-level graphics.